module/API/Measurer.h

```diff
@@ -0,0 +1,92 @@
+/**
+ * @file Measurer.cpp
+ * @brief 計測に用いる関数をまとめたラッパークラス
+ * @author YKhm20020
+ */
+
+#ifndef MEASURER_H
+#define MEASURER_H
+
+#include <algorithm>
+#include "ev3api.h"
+#include "ColorSensor.h"
+#include "SonarSensor.h"
+#include "Motor.h"
+
+class Measurer {
+ public:
+  /**
+   * コンストラクタ
+   */
+  Measurer();
+
+  /**
+   * 明るさを取得
+   * @return 反射光の強さ(0-100)
+   */
+  int getBrightness();
+
+  /**
+   * RGB値を取得
+   * @return RGB値
+   */
+  rgb_raw_t getRawColor();
+
+  /**
+   * 左モータ角位置取得
```

Comment:
```suggestion
   * 左モータ角位置を取得
```
ヘッダーファイル内で「位置を取得」と「位置取得」が統一されていないので、どちらかに統一してください。